fenix/app/src/main/java/org/mozilla/fenix/debugdrawer/ui/DebugDrawerFAB.kt

```diff
@@ -0,0 +1,78 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+package org.mozilla.fenix.debugdrawer.ui
+
+import androidx.compose.animation.AnimatedVisibility
+import androidx.compose.foundation.background
+import androidx.compose.foundation.layout.Column
+import androidx.compose.foundation.layout.Spacer
+import androidx.compose.foundation.layout.height
+import androidx.compose.foundation.layout.padding
+import androidx.compose.runtime.Composable
+import androidx.compose.runtime.getValue
+import androidx.compose.runtime.mutableStateOf
+import androidx.compose.runtime.remember
+import androidx.compose.runtime.setValue
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.res.painterResource
+import androidx.compose.ui.tooling.preview.Preview
+import androidx.compose.ui.unit.dp
+import org.mozilla.fenix.R
+import org.mozilla.fenix.compose.button.FloatingActionButton
+import org.mozilla.fenix.compose.button.PrimaryButton
+import org.mozilla.fenix.theme.FirefoxTheme
+
+/**
+ * Floating Action Button used to reveal the debug drawer.
+ *
+ * @param visible Whether the fab is visible.
+ * @param onClick Invoked when the user clicks on the fab.
+ * @param modifier Modifier to apply to the content.
+ */
+@Composable
+fun DebugDrawerFAB(
+    visible: Boolean,
+    onClick: () -> Unit,
+    modifier: Modifier = Modifier,
+) {
+    AnimatedVisibility(
+        visible = visible,
+        modifier = modifier,
+    ) {
+        FloatingActionButton(
+            icon = painterResource(R.drawable.ic_new),
```

Comment:
we could use a "bug" type icon? 🙂

Reply:
I'm actually working with Crystal to create an icon just for the drawer! [Ticket](https://bugzilla.mozilla.org/show_bug.cgi?id=1866128)